README.md

```diff
@@ -1533,6 +1534,14 @@ The current configuration of the unit tests requires the following environment v
     export NODE_PATH=.
 ````
 
+## Hosted API
+
+We also provide a hosted API for natural which may simplify your use case.
+
+<a href="https://natural.saasify.sh">
```

Comment:
Just tried `https://natural.saasify.sh/`, and it gives an error `The deployment could not be found on ZEIT Now.`